### DISCUSSION TITLE
Help Installing Javinizer

### DISCUSSION THREAD

**Discussion by itsrrm** [javinizer/Javinizer#178] [Q&A]
Hi All,
I just got to the step "Import the module into your shell manually". When I use the command:

`Import-Module .\Javinizer\Javinizer.psm1`

I get the following error:
   
![image](https://user-images.githubusercontent.com/4049570/103820291-15dc0c80-503a-11eb-8fbb-81590d724750.png)

Any assistance on how to proceed from here is appreciated!

**Comment by jvlflame** [comment]
You need to install PowerShell 7 instead of Windows PowerShell.
https://github.com/PowerShell/PowerShell/releases/tag/v7.1.0

**Reply by itsrrm** [reply]
Thank you! That seems to have gotten me a step further, but now I get this when I run "Javinizer -v" to confirm installation.

![image](https://user-images.githubusercontent.com/4049570/103823106-6b66e800-503f-11eb-92e0-665fd6d8b65c.png)

**Reply by jvlflame** [reply]
There shouldn't be a space between the `-` and `-v`. Try it again as `Javinizer -v`.

**Reply by itsrrm** [reply]
Ugh, I feel silly. That was it :) Thanks for the help!

**Reply by itsrrm** [reply]
Hate to have to come back here for more help. I ran the script on a small subset of movies. It looks like it completed successfully, with one warning indicating a movie was skipped as it was a duplicate, which is fine. However, when I go to the destination folder, there's nothing there, it's empty. Furthermore, the source folder is also empty, with the exception of the duplicate movie that was skipped. Thoughts on this? See below the output after running the script, and the parameters I used.

![image](https://user-images.githubusercontent.com/4049570/103825900-b8e55400-5043-11eb-9cd4-b175c9fcaebf.png)

**Reply by jvlflame** [reply]
Javinizer doesn't perform any file deletions, so it's likely your file is still somewhere on your system. When you receive that warning, the source file should not be moved.

If you do `Javinizer -OpenLog`, do you see a log entry of where the file was moved to?

**Reply by itsrrm** [reply]
Thanks. Log shows they were moved to the correct folder. I am pasting the log lines before, I hope that's fine. However, I checked, and the moves are not in the destination folder. I just tried this again with a new set of files, and folders, and it worked well - so I am not sure what may have happened with the below.

[2021-01-06T17:18:49][INFO ] [VEC-447] [Set-JVMovie] Completed [C:\Users\ruddy\Desktop\JAVINIZER\VEC-447.mp4] => [.\JAVINIZER\SORTED\VEC-447 [VENUS] - I Cant Deny My Wifes Best Friend A Creampie Chitose Yuki (2020)\VEC-447.mp4]

[2021-01-06T17:18:49][INFO ] [VEMA-147] [Set-JVMovie] Completed [C:\Users\ruddy\Desktop\JAVINIZER\A\B\VEMA-147.mp4] => [.\JAVINIZER\SORTED\VEMA-147 [VENUS] - I Participated In A 3-Day, 2-Night Creampie School Trip With The Goal Of Fucking My Colossal... (2020)\VEMA-147.mp4]

[2021-01-06T17:18:49][INFO ] [VEMA-147] [Set-JVMovie] Completed [C:\Users\ruddy\Desktop\JAVINIZER\A\B\VEMA-147.mp4] => [.\JAVINIZER\SORTED\VEMA-147 [VENUS] - I Participated In A 3-Day, 2-Night Creampie School Trip With The Goal Of Fucking My Colossal... (2020)\VEMA-147.mp4]

**Reply by itsrrm** [reply]
I'll add: I searched my entire drive for the missing files, and they are not turning up. I would assume they would be somewhere. Thankfully I had created backups, so all good.

**Reply by jvlflame** [reply]
Weird! I'm glad you had backups, but definitely do let me know if you are able to reproduce that issue or run into it again.
Files should definitely not be disappearing into thin air like that so that's pretty worrisome.

**Reply by itsrrm** [reply]
Update here! Turns out the files were NOT deleted after all. I found them buried into my System32 folder. I had created a sort folder there, but was certain I didn't set it as my destination path, but turns out I did. Long story as to why I chose that as my folder, but just thought you should know that all is good! Thanks again!
